src/components/SlateEditor/plugins/embed/SlatePodcast.tsx

```diff
@@ -98,44 +102,61 @@ const SlatePodcast = ({
   return (
     <div draggable {...attributes}>
       <Figure id={`${audio.id}`}>
-        <FigureButtons
-          figureType="podcast"
-          tooltip={t('form.podcast.remove')}
-          onRemoveClick={onRemoveClick}
-          embed={embed}
-          language={language}
-        />
-        <Modal open={editing} onOpenChange={setEditing}>
-          {audio.id ? (
-            <ModalTrigger disabled={!audio.id}>
-              <SlatePodcastWrapper
-                showCopyOutline={showCopyOutline}
-                hasError={hasError}
-                contentEditable={false}
-                role="button"
-                className="c-placeholder-editomode"
-                draggable
-                tabIndex={0}
-                onKeyPress={() => setEditing(true)}
-                onClick={() => setEditing(true)}
-              >
-                <AudioPlayerMounter audio={audio} locale={locale} speech={false} />
-              </SlatePodcastWrapper>
-            </ModalTrigger>
-          ) : (
-            audioLoading && <Spinner />
-          )}
-          <ModalContent>
-            <EditPodcast
-              close={close}
-              embed={embed}
-              locale={locale}
-              podcast={audio}
-              setHasError={setHasError}
-              saveEmbedUpdates={saveEmbedUpdates}
-            />
-          </ModalContent>
-        </Modal>
+        <div>
```

Comment:
Hvorfor er det en div her?

Reply:
For å få linken til å fugnere, den ville ikke fungere når det var contenteditable på div over figure